holidays/countries/fiji.py

```diff
@@ -0,0 +1,128 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import SAT, SUN, OCT, NOV
+from holidays.constants import PUBLIC, WORKDAY
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SUN_TO_NEXT_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+    TUE_TO_PREV_MON,
+)
+
+
+class Fiji(ObservedHolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """
+    References:
+    - https://www.fiji.gov.fj/About-Fiji/Public-Holidays
+    - https://www.timeanddate.com/holidays/fiji/
+    - https://en.wikipedia.org/wiki/List_of_festivals_in_Fiji
+    """
+
+    country = "FJ"
+    weekend = {SAT, SUN}
```

Comment:
`weekend = {SAT, SUN}` is already the default behaviour so we can safely remove this as well.
```suggestion
```